app/mailers/contact_mailer.rb

```diff
@@ -1,17 +1,17 @@
 class ContactMailer < ActionMailer::Base
-  default from: "The Odin Project <admin@theodinproject.com>"
-  FEEDBACK_EMAIL = "feedback@theodinproject.com"
-  CONTACT_EMAIL = "contact@theodinproject.com"
-  ODIN_URL = "http://www.theodinproject.com"
+  default from: 'The Odin Project <admin@theodinproject.com>'
 
-
-  def suggestion_email(suggestion_text, url="", user_email="")
+  def suggestion_email(suggestion_text, url='', user_email='')
```

Comment:
Its possible for a non logged in user to send an suggestion email, The url one I'm not so sure about it. It's the url of the page the user sends the suggestion from. So there should theoretically always be a url.